lib/ext/record.rb

```diff
@@ -55,4 +55,63 @@ def lookup_provider(include_address = nil)
     return { 'npis' => [provider.npi], 'tins' => [provider.tin], 'addresses' => addresses } if include_address
     { 'npis' => [provider.npi], 'tins' => [provider.tin] }
   end
+
+  def duplicate_randomization(random: Random.new)
+    rec = clone
+    rec = randomize_record_name_or_birth(rec, random: random)
```

Comment:
Two lines above can be chained:
`rec = clone.randomize_record_name_or_birth(rec, random: random)`